### PR TITLE
Clean cosmos_qa

### PR DESCRIPTION
**Pull request by nihalnayak**
## Summary of changes
add: accuracy and template description for cosmos_qa. 
add: additional prompt for question generation.

The prompts seemed okay. This is a priority 1 dataset.

### CODE REVIEW

**Comment by awebson** [comment]
Should `Answer Choices` be separated by `|||` not commas?

**Comment by stephenbach** [comment]
Yes, that's right. Separate with `|||`

**Review by awebson — CHANGES_REQUESTED** [review]
@nihalnayak can you double check the original task flags? For example, isn't context_description_question_answer_text the original task of QA?

**Comment by nihalnayak** [comment]
> @nihalnayak can you double check the original task flags? For example, isn't context_description_question_answer_text the original task of QA?
Thanks for flagging this issue. Should I convert this to an original task even though this is a generation task? Should the metric be accuracy instead of ROUGE/BLEU?

**Comment by awebson** [comment]
No you can just leave the generation templates as they are, but just uncheck their "original task" boxes. Meanwhile, for the QA templates, check their "original task" boxes. Thanks!

**Review by awebson — CHANGES_REQUESTED** [review]
Could you please also add the `answer_choices_keys` according to #373 ?

File: promptsource/templates/cosmos_qa/templates.yaml [hunk 1]
```diff
@@ -62,13 +29,15 @@ templates:
       _do_eval: false
       _do_train: true
       choices_in_prompt: true
-      metrics: []
-      original_task: null
+      metrics:
+      - BLEU
```
Comment (awebson) [inline promptsource/templates/cosmos_qa/templates.yaml:33]:
```suggestion
      - Accuracy
```

File: promptsource/templates/cosmos_qa/templates.yaml [hunk 2]
```diff
@@ -62,13 +29,15 @@ templates:
...
+      - ROUGE
```
Comment (awebson) [inline promptsource/templates/cosmos_qa/templates.yaml:34]:
```suggestion
```

File: promptsource/templates/cosmos_qa/templates.yaml [hunk 3]
```diff
@@ -169,17 +141,19 @@ templates:
       _do_eval: false
       _do_train: true
       choices_in_prompt: true
-      metrics: []
-      original_task: null
+      metrics:
+      - BLEU
```
Comment (awebson) [inline promptsource/templates/cosmos_qa/templates.yaml:145]:
```suggestion
      - Accuracy
```

File: promptsource/templates/cosmos_qa/templates.yaml [hunk 4]
```diff
@@ -169,17 +141,19 @@ templates:
...
+      - ROUGE
```
Comment (awebson) [inline promptsource/templates/cosmos_qa/templates.yaml:146]:
```suggestion
```

File: promptsource/templates/cosmos_qa/templates.yaml [hunk 5]
```diff
@@ -470,7 +416,24 @@ templates:
       _do_eval: false
       _do_train: true
       choices_in_prompt: true
-      metrics: []
-      original_task: null
+      metrics:
+      - BLEU
```
Comment (awebson) [inline promptsource/templates/cosmos_qa/templates.yaml:420]:
```suggestion
      - Accuracy
```

File: promptsource/templates/cosmos_qa/templates.yaml [hunk 6]
```diff
@@ -470,7 +416,24 @@ templates:
...
+      - ROUGE
```
Comment (awebson) [inline promptsource/templates/cosmos_qa/templates.yaml:421]:
```suggestion
```

**Review by awebson — APPROVED** [review]

**Pull request merged**